### ISSUE TITLE
NuGet tries to repeatedly authenticate even when authentication fails or times out for one of the authentication feeds.

### ISSUE THREAD

**Issue by anangaur** [NuGet/Home#5373]
In a multi-feed environment, NuGet PM, restore pops up the authentication window everytime in following cases:
1. VPN not connected hence the feed is unreachable
2. The user does not have permissions to access the feed  (there is no feedback to user to understand this?)

**Comment by mishra14** [comment]
@anangaur can you please add this to the common issue.

**Comment by nkolev92** [comment]
@anangaur 

What's the bug here? What should be fixed?

I'd say attempting to authenticate consistently is the expected behavior?
NuGet tries a maximum of 3 times per source!
Note that in the background these sources might use the same token and the user would be prompted only once, but that's beyond NuGet's control.

Unless there's something where we pop up a window after a user has successfully entered their credentials.

**Comment by anangaur** [comment]
This is an old issue where the pop up window to authenticate kept popping. I think this can be closed as we fixed a couple of issues in this area especially around VS pop up to authenticate. We can reopen or open another issue based on new customer input.

**Comment by nkolev92** [comment]
Sounds reasonable. 

Thanks.
